src/ray/object_manager/object_manager.h

```diff
@@ -299,6 +299,8 @@ class ObjectManager : public ObjectManagerInterface,
     return static_cast<double>(used_memory_) / config_.object_store_memory;
   }
 
+  bool PullManagerAtCapacity() const { return pull_manager_->AtCapacity(); }
```

Comment:
I think this should just be checking if num active bundles != num queued bundles.

Reply:
I want to check this more precisely because this can also happen if we don't have object sizes yet.

Reply:
So the condition would be:

`num_active != num_queued - num_missing_object_sizes`? This seems nontrivial to calculate.

I would just use the simpler condition of `num_active != num_queued` for now, since object sizes missing is a very brief transient condition that we quickly move out of.

Reply:
Hmm, my plan was to just recompute it every time we call `UpdatePulls`, but I guess worst case if we have a false positive it just means a delay in spillback. So I'll change it to check the sizes but will rename to "PullsQueued".